CONTRIBUTING.md

```diff
@@ -13,15 +13,15 @@ Contributing code to snap is a snap (pun intended).
 - Push your commit(s) to your repository
 - Open a pull request against the original repo and follow the pull request guidelines below
 
-The maintainers of the repo utilize a "Looks Good To Me" (LGTM) message in the pull request.
+The maintainers of the repo utilize a "Looks Good To Me" (LGTM) message in the pull request. After one or more maintainer states LGTM, we will merge. If you have questions or comments on your code, feel free to correct these in your branch through new commits.
 
 ### Commit Guidelines
 
 Commits should have logical groupings. A bug fix should be a single commit. A new feature
-should be a single commit. 
+should be a single commit.
```

Comment:
An alternative is to ask people to tag issues in the PR instead (granted, issue numbers wouldn't get into the branch history, but those issue numbers are meaningless outside the context of GitHub anyway).